src/shared/common/ScaleBar.js

```diff
@@ -70,19 +83,20 @@ export default (props: Props) => {
             break;
     }
 
-    const p = getScaleBar(meters, feet, tileWidth);
+    const referenceSize = useScreenWidth
+        ? GLOBAL.SCREEN_WIDTH
+        : GLOBAL.TILE_SIZE;
```

Comment:
okay I think this relates to my question above

Reply:
yup :)